ethereum-consensus/src/capella/block_processing.rs

```diff
@@ -252,16 +255,113 @@ pub fn process_withdrawals<
         BYTES_PER_LOGS_BLOOM,
         MAX_EXTRA_DATA_BYTES,
     >,
-    _execution_payload: &ExecutionPayload<
+    execution_payload: &ExecutionPayload<
         BYTES_PER_LOGS_BLOOM,
         MAX_EXTRA_DATA_BYTES,
         MAX_BYTES_PER_TRANSACTION,
         MAX_TRANSACTIONS_PER_PAYLOAD,
         MAX_WITHDRAWALS_PER_PAYLOAD,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<()> {
-    unimplemented!()
+    let expected_withdrawals = get_expected_withdrawals(state, context);
+
+    if execution_payload.withdrawals.as_ref() == expected_withdrawals {
```

Comment:
looks like we got a little carried away in the edit here...
```suggestion
    if execution_payload.withdrawals.as_ref() != expected_withdrawals {
```

Reply:
oh man.. smh